schema/CONDUIT_EARMARK_OUTS.json

```diff
@@ -0,0 +1,779 @@
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 20,
+            "pattern": "^[ -~]{0,20}$",
+            "fec_spec": {
+                "COL_SEQ": 30,
+                "FIELD_DESCRIPTION": "BENEFICIARY CANDIDATE MIDDLE NAME",
+                "TYPE": "A/N-20",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P | F3L"
+            }
+        },
+        "beneficiary_candidate_prefix": {
+            "title": "BENEFICIARY CANDIDATE PREFIX",
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 10,
+            "pattern": "^[ -~]{0,10}$",
+            "fec_spec": {
+                "COL_SEQ": 31,
+                "FIELD_DESCRIPTION": "BENEFICIARY CANDIDATE PREFIX",
+                "TYPE": "A/N-10",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P | F3L"
+            }
+        },
+        "beneficiary_candidate_suffix": {
+            "title": "BENEFICIARY CANDIDATE SUFFIX",
+            "description": "",
+            "type": [
+                "string",
+                "null"
+            ],
+            "minLength": 0,
+            "maxLength": 10,
+            "pattern": "^[ -~]{0,10}$",
+            "fec_spec": {
+                "COL_SEQ": 32,
+                "FIELD_DESCRIPTION": "BENEFICIARY CANDIDATE SUFFIX",
+                "TYPE": "A/N-10",
+                "REQUIRED": null,
+                "SAMPLE_DATA": null,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P | F3L"
+            }
+        },
+        "beneficiary_candidate_office": {
+            "title": "BENEFICIARY CANDIDATE OFFICE",
+            "description": "",
+            "type": "string",
+            "minLength": 0,
+            "maxLength": 1,
+            "pattern": "^[ -~]{0,1}$",
+            "examples": [
+                "H"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 33,
+                "FIELD_DESCRIPTION": "BENEFICIARY CANDIDATE OFFICE",
+                "TYPE": "A/N-1",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "H",
+                "VALUE_REFERENCE": "H,S,P",
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": "F3 | F3X | F3P | F3L"
+            }
+        },
+        "beneficiary_candidate_state": {
+            "title": "BENEFICIARY CANDIDATE STATE",
+            "description": "",
+            "type": [
+                "string",
+                "null"
```

Comment:
`"type": "string"` because it is a required field

Reply:
:+1: